internal/discord/discord.go

```diff
@@ -2,14 +2,19 @@ package discord
 
 import (
 	"fmt"
+	"sync"
 
 	"github.com/bwmarrin/discordgo"
 )
 
 type Discord struct {
 	Token               string
 	Session             *discordgo.Session
+	Ready               sync.WaitGroup
 	ModLoggingChannelID string
+
+	// Roles[guild_id][role_name]
```

Comment:
i almost mistook this for actual code, let's add some more wording to this comment